Resources/doc/operations.md

```diff
@@ -21,8 +21,7 @@ By default, the following operations are automatically enabled:
 ## Disabling operations
 
 If you want to disable some operations (e.g. the `DELETE` operation), you must register manually applicable operations using
-the operation factory class, `Dunglas\ApiBundle\Resource::addCollectionOperation()` and `Dunglas\ApiBundle\Resource::addCollectionOperation()`
-methods.
```

Comment:
`addItemOperation ` doesn't exist in my project, besides, doc speaks about the factory creating operations. Maybe I am lost ^_^